charts/dslorchestrator/values.yaml

```diff
@@ -457,6 +458,8 @@ dslorchestrator:
       mountPathForScenario: "/home/mosip/mountvolume/"
       packetUtilityBaseUrl: https://<server-host>/v1/packetcreator
       s3-account: "dslreports"
-      THREAD_COUNT: '8'
-      push-reports-to-s3: 'yes'
+      threadCount: "8"
+      push-reports-to-s3: "yes"
+      enableDebug: "yes"
+      langselect: "0"
   secrets:
```

Comment:
@Mohanraj209  please keep this pr on hold as these changes have not yet moved to the 1.2.0.1 branch of the mosip-automation-tests repo